scripts/extension-version.test.js

```diff
@@ -161,57 +161,57 @@ describe('next extension version', () => {
     ).to.eq('2.0.2')
   })
 
-  it('it should work with unstable channel after the ga versioning scheme and current version to new version schema', () => {
+  it('it should work with insider channel after the ga versioning scheme and current version to new version schema', () => {
     expect(
       nextExtensionVersion({
-        prismaVersion: '2.0.1-dev.1',
+        prismaVersion: '2.0.1-insider.1',
```

Comment:
The Prisma version naming is not influenced by this.

Reply:
Isn't it confusing though if the Displayname is called Insider and the actual package name is not?

Reply:
As far as I know `prismaVersion` refers to the version name on npm of the Prisma CLI. This is input to the function that calculates the extension version.